src/api/cloud/cloud.aapi.service.js

```diff
@@ -8,6 +8,11 @@ angular.module('ovh-api-services').service('OvhApiCloudAapi', ($resource, $cache
       serviceType: 'aapi',
       cache: queryCache,
     },
+    getDeals: {
+      method: 'GET',
+      url: '/public-cloud-deals',
```

Comment:
To keep consistency across endpoint definition I would recommend you to keep endpoint prefix which is: `/cloud/instances/`

Reply:
I agree to keep `/cloud` but not `/instances` as it doesn't impact only instances.
Does `/cloud/deals` suits you ?

Reply:
LGTM